packages/welldone-wallet/src/lib/welldone.ts

```diff
@@ -360,8 +361,9 @@ export function setupWelldoneWallet({
   deprecated = false,
 }: WelldoneWalletParams = {}): WalletModuleFactory<InjectedWallet> {
   return async () => {
-    const mobile = isMobile();
-    if (mobile) {
+    const support =
+      !isMobile() && isSupportBrowser(["chrome", "edge-chromium", "opera"]);
```

Comment:
Why is it necessary to hide WELLDONE in other browsers?

We intentionally show them (injected wallets) in all browsers because it indicates that the `dApp` has support for that wallet, if you click the WELLDONE option in other browsers the modal will show you the `WalletNotInstalled` view.

Reply:
@kujtimprenkuSQA 

So far, it is not a good user experience when user sees a browser that doesn't support shows up and it redirects to a wrong/irrelevant page. (For example, Welldone wallet, if a user click them during using safari or firefox browser, it actually redirect user to a chrome extension download page. For the sake of simplify, we should only show wallets that are supported on that particular browser. (In case if any wallet teams want to support their wallet visible to many other browsers, they should work on plugin for particular browsers to support them natively)

Do you have different opinion to this? or alternative solution to above problem?

Reply:
> Do you have different opinion to this? or alternative solution to above problem?

No, I don't have any other suggestion, based on what we discussed yesterday we will work on showing wallets only on the browsers they are supported on.